lua/lazyvim/plugins/extras/lang/sql.lua

```diff
@@ -0,0 +1,71 @@
+local sql_ft = { "sql", "mysql", "plsql" }
+
+return {
+  {
+    "nvim-treesitter/nvim-treesitter",
+    optional = true,
+    opts = function(_, opts)
+      if type(opts.ensure_installed) == "table" then
+        vim.list_extend(opts.ensure_installed, { "sql" })
+      end
+    end,
+  },
+  {
+    "folke/edgy.nvim",
+    optional = true,
+    opts = function(_, opts)
+      table.insert(opts.right, {
+        title = "Database",
+        ft = "dbui",
+        pinned = true,
+        open = function()
+          vim.cmd("DBUI")
+        end,
+      })
+
+      table.insert(opts.bottom, {
+        title = "DB Query Result",
+        ft = "dbout",
+      })
+    end,
+  },
+  {
+    "tpope/vim-dadbod",
+    cmd = { "DBUI", "DBUIToggle", "DBUIAddConnection", "DBUIFindBuffer" },
+    dependencies = {
+      "kristijanhusak/vim-dadbod-ui",
+      { "kristijanhusak/vim-dadbod-completion", ft = sql_ft },
+      { "jsborjesson/vim-uppercase-sql", ft = sql_ft },
+    },
+    init = function()
+      vim.g.db_ui_save_location = vim.fn.stdpath("data") .. "/db_ui"
+      vim.g.db_ui_use_nerd_fonts = true
+      vim.g.db_ui_execute_on_save = false
```

Comment:
why `db_ui_execute_on_save` = false by default? 

pardon my lack of knowledge on how dadbod works but how would we run the query otherwise? 😅

Reply:
I found that running on save can lead to Neovim crashing mistakes, for example, if you forget to put a `LIMIT 100` at the end of the query and your table is massive.

So I use the key bind, which are only applied to when this plugin is active.

Leader Shift+s to run the current buffer's query.

Is worth learning also Leader Shift+w as well to save the queries to disk.

Reply:
Yeah manually triggering query is better . This will make sure we are sure about the query . This will shine when you are writing mutations

Reply:
should we add a comment here on the keymap/command to execute the buffer's query?

Reply:
@kevinrobayna hmm yeah, let me see, not only the comment but which key shows something like this:

`S ➜ <Plug>(DBUI_ExecuteQuery)`

I can try to add those descriptions without breaking anything since this only applies to the plugin, probably by using keys, let me give it shot.

Reply:
ah! i see just saw it here! 
![image](https://github.com/LazyVim/LazyVim/assets/5471469/69432560-538d-48bf-9f53-e92ae84ca36f)